Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -540,6 +548,9 @@ void WebProcessProxy::platformGetLaunchOptions(ProcessLauncher::LaunchOptions& l
 
 bool WebProcessProxy::shouldSendPendingMessage(const PendingMessage& message)
 {
+    // Send the quirk before handling any load requests.
+    sendStorageAccessUserAgentStringQuirksData();
```

Comment:
Seems like an odd place to do this. Could you explain why calling this in didFinishLaunching() does not suffice? I'd expect didFinishLaunching() to be one of the first thing to get called.

Maybe you just need to call `sendStorageAccessUserAgentStringQuirksData()` inside `didFinishLaunching()` BEFORE the call to `AuxiliaryProcessProxy::didFinishLaunching()`, not after.